spec/features/cart/show_spec.rb

```diff
@@ -4,12 +4,12 @@
   describe 'When I have added items to my cart' do
     describe 'and visit my cart path' do
       before(:each) do
-        @mike = Merchant.create(name: "Mike's Print Shop", address: '123 Paper Rd.', city: 'Denver', state: 'CO', zip: 80203)
-        @meg = Merchant.create(name: "Meg's Bike Shop", address: '123 Bike Rd.', city: 'Denver', state: 'CO', zip: 80203)
+        @mike = Merchant.create!(name: "Mike's Print Shop", address: '123 Paper Rd.', city: 'Denver', state: 'CO', zip: 80203)
```

Comment:
It took a while, but we did get them all, and it was a good idea.

Reply:
I'm glad we learned so much on this user story and set of stories. Shopping cart was intense, but we bonded and learned!!!